src/standup/standup.service.ts

```diff
@@ -1,63 +1,96 @@
+import {
+  AstraService,
+  deleteItem,
+  documentId,
+} from '@cahllagerfeld/nestjs-astra';
 import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
 import { StandupDTO } from './dto/standup.dto';
 import { Standup } from './interfaces/standup.interface';
+import { concatMap, filter } from 'rxjs/operators';
 
 @Injectable()
 export class StandupService {
-  private standups: Array<Standup> = [];
+  constructor(private readonly astraService: AstraService) {}
 
-  create(body: StandupDTO): Standup {
+  create(body: StandupDTO) {
     const { discordUser, todayMessage, yesterdayMessage } = body;
 
     const newStandup: Standup = {
-      id: 123,
       yesterdayMessage: yesterdayMessage,
       todayMessage: todayMessage,
       discordUser: discordUser,
-      createdOn: new Date('2021-01-01T00:00:00.000Z'),
+      createdOn: new Date(Date.now()),
```

Comment:
I think this can be ...
```suggestion
      createdOn: new Date(),
```